dm/tests/requirements.txt

```diff
@@ -1 +1,2 @@
 requests
+ipaddress
```

Comment:
where do we add this python dependency? I don't see the changes in this PR

Reply:
integration test will install in Makefile:[install dependecy from make](https://github.com/pingcap/tiflow/blob/master/Makefile#L494-L496)

Reply:
Sorry I mean where's the Python code uses this package? Because it's a newly added dependency, I think this PR should also modify a Python script that newly uses it.